src/components/DataTable.js

```diff
@@ -83,7 +83,7 @@ export default class DataTable extends React.Component {
           <tbody>
             {values.map((val, i) => (
               <tr key={i}>
-                {columns.map((col, j) => <td key={j}>{wide ? '' : col + ':'} {val[col]}</td>)}
+                {columns.map((col, j) => <td key={j}><span>{wide ? '' : col + ':'}</span> {val[col]}</td>)}
```

Comment:
kinda weird that this is an empty span when the table is wide? i think

```
{wide ? '' : <span>{col}:</span>}
```

would work? maybe, i'm not sure about jsx inside ternaries